app/managers.py

```diff
@@ -0,0 +1,54 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.table_name = "actors"
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+        with self._connection:
+            self._connection.execute(
+                f"CREATE TABLE IF NOT EXISTS {self.table_name} ("
+                "id INTEGER PRIMARY KEY, "
+                "first_name VARCHAR(50) NOT NULL, "
+                "last_name VARCHAR(50) NOT NULL"
+                ")"
+            )

```

Comment:
I thought we have to create the table from DB sql window.

Reply:
@ORENYT  It wasn't mentioned